pkg/threescale/backend/manager.go

```diff
@@ -44,13 +44,17 @@ type LimitCounter map[string]map[api.Period]*Limit
 
 // cachedApp provides a placeholder for information for remote reporting
 type cachedApp struct {
+	// snapshot stores the app state (copy) as it was when we attempted to flush the cache
 	snapshot     Application
 	cacheKey     string
 	serviceID    api.Service
 	appID        string
 	reportingErr bool
 	authErr      bool
 	authResp     *threescale.AuthorizeResult
+	// deltas stores the metrics (copy) that were actually reported to 3scale after
```

Comment:
very minor thing: typos in the commit message `metirc` and `stroe`

Reply:
fixed

Reply:
fixed